setup.cfg

```diff
@@ -40,55 +41,55 @@ install_requires =
 [options.extras_require]
 # The list of available extras is also in the installation docs, if you add or remove one please edit it there as well.
 asdf =
-  asdf
+  asdf>=2.6.0
 dask =
-  dask[array]
+  dask[array]>=2.0.0
 database =
-  sqlalchemy
+  sqlalchemy>=1.3.4
 instr =
-  matplotlib>=3.1.1
+  matplotlib>=3.1.0
   pandas>=0.24.0
   scipy>=1.3.0
 image =
-  scikit-image
+  scikit-image>=0.15.0
   scipy>=1.3.0
 jpeg2000 =
-  glymur!=0.9.0
+  glymur>=0.8.18,!=0.9.0
 map =
-  matplotlib>=3.1.1
+  matplotlib>=3.1.0
   scipy>=1.3.0
 net =
-  beautifulsoup4
-  pandas>=0.24.0
+  beautifulsoup4>=4.8.0
   drms>=0.6.1
-  python-dateutil
-  tqdm
-  zeep
+  pandas>=0.24.0
+  python-dateutil>=2.8.0
+  tqdm>=4.32.1
+  zeep>=3.4.0
 timeseries =
   h5netcdf>=0.8.1
-  matplotlib>=3.1.1
+  matplotlib>=3.1.0
   pandas>=0.24.0
 visualization =
-  matplotlib>=3.1.1
+  matplotlib>=3.1.0
 tests =
-  pytest-astropy >= 0.8  # 0.8 is the first release to include filter-subpackage
-  pytest-doctestplus >= 0.5 # We require the newest version of doctest plus to use +IGNORE_WARNINGS
+  jplephem  # For some coordinates tests
+  pytest-astropy
+  pytest-doctestplus
   pytest-mock
-  pytest-mpl >= 0.12
```

Comment:
why?

Reply:
I can add them back? I don't think it really matters for the test deps.

Reply:
Well if we rely on a feature in a particular release we should tell pip about that when people are installing so it auto upgrades things if they are already installed.